consensus/hotstuff_replica.go

```diff
@@ -136,6 +192,12 @@ func (handler *HotstuffReplicaMessageHandler) HandleDecideMessage(m *consensusMo
 	}
 
 	m.paceMaker.NewHeight()
+	m.GetBus().
```

Comment:
If you move this into `NewHeight`, then we don't need to call the same counter increment in both the leader & replica code